### PR TITLE
Scalapack 2.0.2 (fix)

### PR DESCRIPTION
**Pull request by davydden**
avoid building dynamic libraries due to a segmentation fault.

### CODE REVIEW

**Comment by dpo** [comment]
Interesting. Thanks for the fix! There are just a few audit issues:
```
Error: 5 problems in 1 formula
homebrew/science/scalapack:
 * Stable: SHA1 checksums are deprecated, please use SHA256
 * 24: Trailing whitespace was found
 * 27: Trailing whitespace was found
 * 28: Trailing whitespace was found
 * 30: Trailing whitespace was found
```

**Comment by dpo** [comment]
Also if you don't mind adding a `desc`...

**Comment by davydden** [comment]
pls merge it before https://github.com/Homebrew/homebrew-science/pull/2630 so that the bottles of petsc are build against static scalapack.

File: scalapack.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class Scalapack < Formula
+  desc "library of high-performance linear algebra routines for parallel distributed memory machines"
   homepage "http://www.netlib.org/scalapack/"
   url "http://www.netlib.org/scalapack/scalapack-2.0.2.tgz"
-  sha1 "ff9532120c2cffa79aef5e4c2f38777c6a1f3e6a"
+  sha256 "0c74aeae690fe5ee4db7926f49c5d0bb69ce09eea75beb915e00bba07530395c"
   head "https://icl.cs.utk.edu/svn/scalapack-dev/scalapack/trunk", :using => :svn
   revision 2
```
Comment (dpo) [inline scalapack.rb]: Sorry, the revision number must be bumped.

**Comment by davydden** [comment]
@dpo i am sorry to say, but this PR was not a solution. Apparently it's `(p)arpack problem` (see https://github.com/opencollab/arpack-ng/issues/18#issuecomment-135168310 ). 

Moreover making the library static broke `mumps` installation on linuxbrew. Could you please undo the PR. 

Sorry again for inconvenience.

**Pull request closed without merging**